src/app/shared/components/user-detail-page/user-overview/user-overview.component.ts

```diff
@@ -0,0 +1,61 @@
+import {Component, HostBinding, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {UsersService} from '../../../../core/services/api/users.service';
+import {MenuItem} from '../../../models/MenuItem';
+import {User} from '../../../../core/models/User';
+import {UserDataService} from '../../../../core/services/common/user-data.service';
+
+@Component({
+  selector: 'app-user-overview',
+  templateUrl: './user-overview.component.html',
+  styleUrls: ['./user-overview.component.scss']
+})
+export class UserOverviewComponent implements OnInit {
+
+  @HostBinding('class.router-component') true;
+
+  constructor(
+    private route: ActivatedRoute,
+    private usersService: UsersService,
+    private userDataService: UserDataService
+  ) { }
+
+  navItems: MenuItem[] = [];
+  user: User;
+  path: string;
+
+  ngOnInit() {
+    this.user = this.userDataService.user;
+    this.path = this.userDataService.path;
+    this.initNavItems();
+  }
+
+  private initNavItems() {
+    this.navItems = [
+      {
+        icon: 'user-white.svg',
+        url: `${this.path}/detail`,
```

Comment:
Actually, in all of these cases, you can just use relative paths instead of absolute paths. 
For example:
```suggestion
        url: 'detail',
```